README.md

```diff
@@ -20,6 +20,13 @@ was written correctly and much more.
 
 ![Etcher](https://raw.githubusercontent.com/resin-io/etcher/master/screenshot.png)
 
+Supported Operating Systems
+---------------------------
+
+- GNU/Linux
+- macOS 10.9 and later
```

Comment:
Oh, thanks for clarifying, I guess the should update [their website](http://electron.atom.io/docs/development/build-instructions-osx/) then